library/Fission/IPFS/Client.hs

```diff
@@ -42,6 +42,7 @@ cat   :: Text            -> ClientM File.Serialized
 pin   :: Text            -> ClientM Pin.Response
 unpin :: Text -> Bool     -> ClientM Pin.Response
 
+-- QUESTION: WHere the hell does this get called?
```

Comment:
Here's an example use: https://github.com/fission-suite/web-api/blob/master/library/Fission/CLI/Pin.hs#L43

Reply:
(the `IPFS.Client.pin hash` part)